test/pseudo-tty/ref_keeps_node_running.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+require('../common');
+
+const TTY = process.binding('tty_wrap').TTY;
+const isTTY = process.binding('tty_wrap').isTTY;
+const strictEqual = require('assert').strictEqual;
+
+strictEqual(isTTY(0), true, 'fd 0 is not a TTY');
```

Comment:
It doesn't for me. I replaced the `strictEqual(isTTY(0), ...)` line with `console.log(isTTY(0))` and I get `false` output but no failure output and a success exit code when I pipe something in (such as `echo -n`). So I think this assertion is valuable. 😉